test/client/recordUtil.js

```diff
@@ -370,7 +370,20 @@ test('recordUtil.resolve', (t) => {
 })
 
 test('recordUtil.resolveRecords()', (t) => {
-  t.plan(4)
+  t.plan(5)
+
+  t.test(`${t.name} resolves same data cross-platform on laptop and android`, (t) => {
+    t.plan(1)
+    const androidRecordsAndExistingObjects = require('./data/resolveAndroid').data
```

Comment:
Minor: It may help to add a comment explaining how the `./data/resolveAndroid` and `./data/resolveLaptop` fixtures had been generated, such that someone could go and regenerate them.